packages/strapi-design-system/src/JSONInput/JSONInput.stories.mdx

```diff
@@ -0,0 +1,99 @@
+<!--- JSONInput.stories.mdx --->
+
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
```

Comment:
This now lives in `docs/stories` 👍🏼

Reply:
@markkaylor this is most likely why your E2E test is failing.

Reply:
ohh thanks! I'll update